### DISCUSSION TITLE
笔记写作的反馈不足

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#68] [主发布站的评论系统]
# 笔记写作的反馈不足

%%我的理解:: 任何努力，有反馈才能帮助我们不断改进，避免闭门造车，做笔记也是如此。所以[[数字花园]]这种形式很吸引我，能让我的笔记被看到。但现在由于缺少积累，很少人给我反馈，哎。%%
  Thoughts Memo 汉化组译制
 原文： Note-writing practices provide weak feedback (andymatuschak.org)
[[笔记写作实践大多是无效的]]，其中一个原因可能是由于笔记系统一般提供的反馈不足。
开始进行间隔重复练习的人每天都会得到强反馈：如果他写了一个糟糕的问题，这个问题便会立刻且此后长期地困扰着他；他会感觉到（在某种程度上）自己对某一主题的记忆保留率的增长。
相比之下，笔记写作的反馈非常迟缓：通常，当你写了一个笔记，你可能在几周内都不会去看它。这样的反馈也模棱两可：如果一个笔记能（或不能）帮助你提炼出一些深刻见解，它通常都不会特别明显。
一般来说，人们并不清楚一个笔记应该是什么样子，所以也就不清楚某个笔记是否符合这个标准。
[[常青笔记]]提供了更紧密和更强的反馈回路，这是因为你在不断地修改和参考过去的笔记。如果之前的笔记难以修改，你也会更容易注意到。使用[[可执行的写作策略]]也会对一个人的笔记系统产生更强的反馈。

https://garden.oldwinter.top/%E7%AC%94%E8%AE%B0%E5%86%99%E4%BD%9C%E7%9A%84%E5%8F%8D%E9%A6%88%E4%B8%8D%E8%B6%B3

**Comment by CatsOfMrXue** [comment]
看到博主说很少有反馈，我就来给点感受吧，作为刚接触数字花园这种的形式的读者，我的感觉自己好像从来没有完整读完过一篇文章，由于每篇文章都有很多跳转，大多数情况下都要跳转过去阅读，再返回继续阅读，才能保证很好地理解，但是有时候跳转层次过深会忘记最开始的文章的主题，从读者角度来看，还是线性的阅读会更具有系统性，当然正如博主所说，这些笔记首先是要为自己的思考服务，再是为读者服务。数字花园的形式也带来了一些好处，我可以不限于当前文章的主题，在跳转的途中会碰到了更有意思的主题，更引起了我的好奇心和探索欲

**Reply by oldwinter** [reply]
感谢反馈~~你很精准地说出了我这个数字花园的特点哈哈~说实话，我自己也能感受到这类笔记阅读起来的体验确实不那么好。我也还在探索中。期待将来能借助ai，达到一种，即花费我很少的时间，就输出了知识的目的，又给读者比较不错的阅读体验，这种平衡状态。

**Reply by BigZhangRocky** [reply]
不太清楚是什么情况下需要跳转？个人觉得跳转的原因大多是因为遇到了某个自己不是很熟悉的概念。

数字花园形式应该是成长型的个人笔记，虽然博主公开发布了，但并没有改变其形式，阅读的时候难免要配合发布者的思维脉络。

长文可能更适合读者保持专注，有足够的上下文提供参考（且作者通常都有安排起承转合），不需要在卡片的思维之间跳跃。

一点碎碎念。
